src/test/java/org/lsc/Ldap2LdapHookSyncTest.java

```diff
@@ -155,31 +153,40 @@ public final void testLdap2LdapLDIFHookSyncTest() throws Exception {
 
 	/*
 	 * Read hook log file to check passed arguments and modification passed as input
-	*/
+	 */
 	private final void checkLDIFSyncResults(String operation, List<String> expectedEntry) throws Exception {
 
 		List<String> hookResults = new ArrayList<String>();
 		try {
 			File hookFile = new File("hook-ldif-" + operation + ".log");
-			Scanner hookReader = new Scanner(hookFile);
 
-			while (hookReader.hasNextLine()) {
-				String data = hookReader.nextLine();
-				hookResults.add(data);
-			}
-			hookReader.close();
+            Thread.sleep(1000L);
```

Comment:
Is this mandatory to have Thread.sleep for this test to work?
If so, the test design is a little poor: it would break on systems with a slow disk. But I have currently no better idea.

Reply:
No, I think it should be removed